### PR TITLE
Fix CSS Icons

### PR DESCRIPTION
**Pull request by joshwooding**
Turns out re-exporting all the icons had a downside... The plugin strips the xmlns info which is needed for the CSS icons.

This would have been caught but the qa story didn't have chromatic turned on.

I've improved the story by showing all icons and making sure that chromatic captures it.

I also made sure that saltIcons.css is in the package.json so it gets published.

### CODE REVIEW

**Review by lukeac123 — APPROVED** [review]

**Review by lilyvc — APPROVED** [review]

File: packages/countries/saltCountries.css
Comment (origami-z) [inline packages/countries/saltCountries.css:1]: Why there a change on this file? no build script change on countries package?

**Comment by origami-z** [comment]
Plugin has a bug, it striped `xmlns` by mistake while trying to remove `fill`. I will update

File: packages/countries/saltCountries.css
Comment (joshwooding) [reply packages/countries/saltCountries.css:1]: The build script for icons and country symbols doesn't ignore newlines. So you get different build outputs on Windows. I've made a fix that strips all new lines when encoding the icon.

**Comment by joshwooding** [comment]
Need to look at the different in the status indicator

**Review by pseys — APPROVED** [review]

**Review by origami-z — APPROVED** [review]

**Pull request merged**